react/week4/github-userSearch/src/components/UserList.js

```diff
@@ -0,0 +1,26 @@
+import React, { useContext } from "react";
+import User from "./User";
+import { UserContext } from "./../App";
+
+function UserList() {
+	const { users } = useContext(UserContext);
+	return (
+		<div className="userList">
+			<ul>
+				{users.length > 0 ? (
+					users.map(user => {
+						return (
+							<li className="items">

```

Comment:
You should use `key` on the list item, please remove it from the `<User>`
```suggestion
							<li key={user.id} className="items">

```